sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/StaticFieldInitializerOrder.cs

```diff
@@ -26,5 +30,7 @@ public partial class StaticFieldInitializerOrder
     public partial class StaticFieldInitializerOrder
     {
         public static int W = 2;
+
+        public const int Const2 = 5;
     }
 }

```

Comment:
In order to avoid future issues with partial on different files I would add a test using partial classes into 2 files. Can you handle it? I can show you how to if needed.